runtime/src/iree/vm/CMakeLists.txt

```diff
@@ -70,95 +70,97 @@ iree_cc_library(
   PUBLIC
 )
 
-iree_cc_test(
```

Comment:
We use bazel-to-cmake and need to make changes in the bazel files first - otherwise these will be overwritten the next time we generate them. You can run it with `python build_tools/bazel_to_cmake/bazel_to_cmake.py` and see what survives - some directories don't use it, but most do. Look for the giant block at the top of the file:
```
################################################################################
# Autogenerated by build_tools/bazel_to_cmake/bazel_to_cmake.py from           #
# runtime/src/iree/runtime/testdata/BUILD.bazel                                #
#                                                                              #
# Use iree_cmake_extra_content from iree/build_defs.oss.bzl to add arbitrary   #
# CMake-only content.                                                          #
#                                                                              #
# To disable autogeneration for this file entirely, delete this header.        #
################################################################################
```

There's ways of inserting the conditions (see some of the bazel files with extra content markers).

But, none of this is required - iree_cc_test disables itself unless IREE_BUILD_TESTS is set:
https://github.com/iree-org/iree/blob/63a2d141adf7b3911bc94c7f779e1896fb25aecb/build_tools/cmake/iree_cc_test.cmake#L54-L57